src/Views/DisplaysView.vala

```diff
@@ -44,9 +46,19 @@ public class Display.DisplaysView : Gtk.Grid {
 
             var dpi_label = new Gtk.Label (_("Scaling factor:"));
 
+            int width, height;
+            monitor_manager.virtual_monitors[0].get_current_mode_size (out width, out height);
+
             dpi_combo = new Gtk.ComboBoxText ();
-            dpi_combo.append_text (_("LoDPI"));
-            dpi_combo.append_text (_("Pixel Doubled"));
+            dpi_combo.append_text (_("LoDPI") + " (1×)");
+            int max_scale = height / 768;
```

Comment:
Should probably pull this out into a constant so it's more obvious where it's coming from and what it represents, e.g. `MIN_SUPPORTED_HEIGHT`